libs/git_wrapper/Git_wrapper.ml

```diff
@@ -749,10 +762,29 @@ let cat_file_blob ?cwd (hash : hash) =
   | Error (`Msg s) ->
       Error s
 
-let with_git_repo (files : Testutil_files.t list) func =
-  Testutil_files.with_tempfiles_verbose files (fun path ->
-      Testutil_files.with_chdir path (fun () ->
-          init ();
-          add ~force:true [ Fpath.v "." ];
-          commit "Add all the files";
-          func ()))
+(*****************************************************************************)
+(* Combination of git commands (for testing etc.) *)
+(*****************************************************************************)
+
+let create_git_repo ?(honor_gitignore = true)
+    ?(user_email = "tester@example.com") ?(user_name = "Tester") () =
+  flush stdout;
```

Comment:
why we need those flush?

Reply:
to get our stuff printed before the output from the `git` process.